apps/ui/src/views/Space/Editor.vue

```diff
@@ -214,6 +216,12 @@ async function handleProposeClick() {
       );
     } else {
       const appName = (route.query.app as LocationQueryValue) || '';
+      const currentTime = Math.floor(Date.now() / 1000);
+      const start = currentTime + props.space.voting_delay;
+      const minEnd =
+        start + (props.space.min_voting_period || DEFAULT_VOTING_DELAY);
+      const maxEnd =
+        start + (props.space.max_voting_period || DEFAULT_VOTING_DELAY);
```

Comment:
Do we need to fallback it here?

We didn't fallback it previously (we had fallback for null/undefined values):
https://github.com/snapshot-labs/sx-monorepo/blob/4b45ddd6e4b058c4f2acf87ae26bed077ff918f0/apps/ui/src/networks/offchain/api/index.ts#L178-L179

This fallback will only do anything if `min_voting_period` is 0, not sure if it's intended or not.